doc/source/whatsnew/v1.0.0.rst

```diff
@@ -201,7 +201,7 @@ ExtensionArray
 Other
 ^^^^^
 - Trying to set the ``display.precision``, ``display.max_rows`` or ``display.max_columns`` using :meth:`set_option` to anything but a ``None`` or a positive int will raise a ``ValueError`` (:issue:`23348`)
-
+- Bug in :meth:`pandas.io.formats.style.Styler.default_display_func` because wasn't displaying decimals digits as supposed (:issue:`13257`)
```

Comment:
Sorry, I misread this earlier. `default_display_func` isn't a method on Styler, it's a closure.

Rather, just describe the behavior and change. Something like

```suggestion
- Bug in :meth:`pandas.io.formats.style.Styler` formatting for floating values not displaying decimals correctly (:issue:`13257`)
```